### ISSUE TITLE
Cannot edit post to add alt text to media

### ISSUE THREAD

**Issue by rrgeorge** [feditext/feditext#304]
**Describe the bug**
Trying to edit a post to add alt text to media backs out before you can save the changes

**To Reproduce**
1. Create a post with an attachment
2. Edit the post
3. Edit the alt text of the media
4. Press Done
5. See the edit modal is dismissed without posting the update

**Expected behavior**
To be able to post the update

**(please complete the following information):**
 - Device: iPhone 12 Pro
 - OS: iOS 17.1.1
 - App Version 1.7.2 (78)

**Comment by weirdofhermiston** [comment]
I have this issue too, also can't update previously entered alt text
